src/main/java/org/dspace/pack/bagit/xml/metadata/Value.java

```diff
@@ -48,6 +63,48 @@ public Value(final String body, final String name) {
         this.name = name;
     }
 
+    /**
+     * Gets the {@code MetadataFieldName} for this value.
+     *
+     * @return the appropriate MetadataFieldName for the given field.
+     * @throws IllegalStateException if this value does not have a valid MetadataFieldName.
+     */
+    public MetadataFieldName getMetadataField() {
+        if (schema != null) {
+            return new MetadataFieldName(schema, element, qualifier);
+        }
+        switch (name) {
```

Comment:
It seems like this switch statement would fit better in the MetadataFieldName class (along with the null schema check).

Reply:
I see what you're saying, but `MetadataFieldName` is a core `dspace-api` class rather than a RTS class, so we can't customize it with this PR. In addition, these "legacy" field names have been deprecated and support for them was mostly *intentionally* removed from DSpace 7, via this commit: https://github.com/mwoodiupui/DSpace/commit/e94911ba1909771baa9d700c4e5f591906014269

My limited understanding is that these were old named fields that have since been migrated to proper DSpace metadata values (which have, at least, a schema and a qualifier, like dc.title = name). So the preference is to have clients of DSpace generally using those newer, `schema.element[.qualifier]` names directly rather than the older ones going forward.

*Some* support for them still lives on deep in the core DSpace code, like here: https://github.com/DSpace/DSpace/blob/main/dspace-api/src/main/java/org/dspace/content/packager/PackageUtils.java#L74-L83

And I noticed that the bagit unit tests that were written previously depended on these names being resolvable. But I don't fully understand if they're necessary, to be honest. It seems ideal that we could remove this type of mapping entirely, as that's the direction DSpace code has been going in...but I didn't want to break the tests and didn't understand if the packager, in real life scenarios, really depends on these legacy field names.